app/services/tracks/boundary_detector.rb

```
@@ -0,0 +1,188 @@
+    end
+
+    resolved_count
+  end
+
+  private
+
+  # Find tracks that might span chunk boundaries
+  def find_boundary_track_candidates
+    # Get recent tracks that might have boundary issues
+    recent_tracks = user.tracks
+                       .where('created_at > ?', 1.hour.ago)
```

RuboCop (1.76.1)

[convention] Align `.where` with `.tracks` on line 33.

(Layout/MultilineMethodCallIndentation)